src/components/scenes/SweepPrivateKeySelectCryptoScene.tsx

```diff
@@ -0,0 +1,165 @@
+  const {
+    currencyConfig: { allTokens },
+    currencyInfo: { displayName: mainnetDisplayName, pluginId }
+  } = receivingWallet
+
+  // const sweepPrivateKeyList = React.useMemo<SweepPrivateKeyItem[]>(() => {
+  //   const list: SweepPrivateKeyItem[] = [{ key: 'null', displayName: displayName, pluginId, tokenId: null }]
+  //   for (const [tokenId, bal] of memoryWallet.balanceMap.entries()) {
+  //     if (zeroString(bal) || tokenId == null) continue
+  //     list.push({ key: tokenId, displayName: allTokens[tokenId].displayName, pluginId, tokenId })
+  //   }
+  //   return list
+  // }, [allTokens, displayName, memoryWallet.balanceMap, pluginId])
+
+  const [selectedItems, setSelectedItems] = React.useState<Set<string>>(() => {
+    const out: Set<string> = new Set()
+    for (const sweepPrivateKeyItem of sweepPrivateKeyList) {
+      out.add(sweepPrivateKeyItem.key)
+    }
+    return out
+  })
+
+  React.useEffect(() => {
+    const unsubscribe = navigation.addListener('beforeRemove', ev => {
+      memoryWallet.close().catch(() => {})
+    })
+    return unsubscribe
+  }, [memoryWallet, navigation])
+
+  const numSelected = selectedItems.size
+
+  const handleCreateWalletToggle = useHandler((key: string) => {
+    const copy = new Set(selectedItems)
+    if (selectedItems.has(key)) copy.delete(key)
+    else copy.add(key)
+    setSelectedItems(copy)
+  })
+
+  const handleNext = useHandler(async () => {
+    if (numSelected === 0) {
+      showError(lstrings.create_wallet_no_assets_selected)
+      return
+    }
+
+    const filteredSweepPrivateKeyList = sweepPrivateKeyList.filter(item => selectedItems.has(item.key))
+    navigation.push('sweepPrivateKeyCalculateFee', {
+      memoryWallet,
+      receivingWalletId,
+      sweepPrivateKeyList: filteredSweepPrivateKeyList
+    })
+  })
+
+  const renderCreateWalletRow = useHandler((item: ListRenderItemInfo<SweepPrivateKeyItem>) => {
+    const { key, tokenId } = item.item
+    const displayName = tokenId == null ? mainnetDisplayName : allTokens[tokenId].displayName
+
+    const toggle = (
+      <Switch
+        ios_backgroundColor={theme.toggleButtonOff}
+        trackColor={{
+          false: theme.toggleButtonOff,
+          true: theme.toggleButton
+        }}
+        value={selectedItems.has(key)}
+        onValueChange={() => handleCreateWalletToggle(key)}
+      />
+    )
+
+    return (
+      <CreateWalletSelectCryptoRow
+        pluginId={pluginId}
+        tokenId={tokenId}
+        walletName={displayName}
+        onPress={() => handleCreateWalletToggle(key)}
+        rightSide={toggle}
+      />
+    )
+  })
+
+  const renderNextButton = React.useMemo(
+    () => (
+      <Fade noFadeIn={numSelected > 0} visible={numSelected > 0} duration={300}>
+        <View style={styles.bottomButton}>
+          <MainButton label={lstrings.string_next_capitalized} type="primary" marginRem={[0, 0, 0.75]} onPress={handleNext} />
+        </View>
+      </Fade>
```

Comment:
Just put this inline below, OR make it a normal function that just gets called as part of the render (`{ renderNextButton() }`), instead of a `useMemo`.